source/text/Font.cpp

```diff
@@ -294,50 +307,54 @@ void Font::CalculateAdvances(ImageBuffer &image)
 
 void Font::SetUpShader(float glyphW, float glyphH)
 {
-	glyphW *= .5f;
-	glyphH *= .5f;
+	glyphWidth = glyphW * .5f;
+	glyphHeight = glyphH * .5f;
 
 	shader = GameData::Shaders().Get("font");
 	glUseProgram(shader->Object());
 	glUniform1i(shader->Uniform("tex"), 0);
 	glUseProgram(0);
```

Comment:
Minor nitpick, this also only needs to be done once.

Reply:
Tbh I don't know enough about opengl shaders to know which parts you are referring to

Reply:
Lines 314-316 is also shared between sizes, so only need to be done once (ergo, moved into the if statement as well).